store/iavl/store.go

```diff
@@ -410,28 +410,3 @@ func getProofFromTree(tree *iavl.MutableTree, key []byte, exists bool) *tmcrypto
 	op := types.NewIavlCommitmentOp(key, commitmentProof)
 	return &tmcrypto.ProofOps{Ops: []tmcrypto.ProofOp{op.ProofOp()}}
 }
-
-//----------------------------------------
```

Comment:
I couldn't find anywhere where this is used, and it is private, so isn't breaking to remove.